portworx/templates/portworx-storagecluster.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
     portworx.io/is-eks: "true"
     portworx.com/install-source: helm/{{.Release.Service}}-r{{.Release.Revision}}
     portworx.com/helm-vars: chart="{{.Chart.Name}}-{{.Chart.Version}}"{{range $k, $v := .Values }}{{if $v}},{{ $k }}="{{ $v }}" {{end}}{{end}}
+    portworx.io/is-eks: "true"
```

Comment:
This is already present on line 17

Reply:
yeah that was a merge issue. i removed it and pushed changes.